ccm/lb-svc-production-plan.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: v1
+kind: Service
+metadata:
+  annotations:
+    service.beta.kubernetes.io/upcloud-load-balancer-config: |
+      {
+        "name": "hello-uks-production-lb",
```

Comment:
name can be omitted, in favour of service name, just in case.

Reply:
Will update & merge!